node/src/chain_spec.rs

```diff
@@ -45,7 +50,9 @@ pub fn dev(runtime: Option<Vec<u8>>) -> ChainSpec {
     .into_chain_spec()
 }
 
-pub fn devnet(runtime: Option<Vec<u8>>) -> ChainSpec {
+/// Chain that is running on the cloud, frequently reset, and might not work but always uses the
+/// latest code.
```

Comment:
I would like to be explicit here. I read it and wondered if it was referring to the native code.

```suggestion
/// latest wasm code.
```